### DISCUSSION TITLE
Joint ICTP-IAEA Workshop on Monte Carlo 2024

### DISCUSSION THREAD

**Discussion by ftessier** [nrc-cnrc/EGSnrc#1097] [Announcements]
**Registration is now open** for the 2024 edition of the [Joint ICTP-IAEA Workshop on Monte Carlo Radiation Transport and Associated Data Needs for Medical Applications](https://indico.ictp.it/event/10516) to be held in Trieste, Italy, from 28 October 2024 to 8 Novembre 2024:

> The Workshop constitutes a unique opportunity for scientists, engineers and medical physicists to gain extensive and up-to-date training on the use and understanding of Monte Carlo methods of relevance to medical applications in therapy and diagnostics, in particular in the use of the state-of-the-art BEAMnrc/EGSnrc Monte Carlo code. Also, participants will be introduced to the on-line data retrieval for medical applications.

> **Grants:** A limited number of grants are available to support the attendance of selected participants, with _priority given to participants from developing countries. There is no registration fee._

**Comment by leylex** [comment]
Hi! Do you know when the results of this application will be published? Thank you!

**Reply by leylex** [reply]
@ftessier @rtownson Hi, I hope you can answer this. Thank you!

**Reply by ftessier** [reply]
We do not handle the registration process, so you must check the ICTP portal. All I know is that the final selection is imminent.
